packages/fury-adapter-oas3-parser/lib/parser/annotations.js

```diff
@@ -38,6 +38,10 @@ function createMemberValueNotBooleanError(namespace, path, member) {
 }
 
 function validateObjectContainsRequiredKeys(namespace, path, requiredKeys, object) {
+  if (!isObject(object)) {
```

Comment:
Since `validateObjectContainsRequiredKeys` can sometimes be ran before the `isObject` validation it would fail we need protection against it.

This is an interim patch because I was planning to add `requiredKeys` behaviour to `parseObject` itself.